admin_guide/sysctls.adoc

```diff
@@ -0,0 +1,168 @@
+requirement for sysctls to be accessible in a pod context within Kubernetes.
+
+The following sysctls are known to be namespaced:
+
+- *_kernel.shm*_*
+- *_kernel.msg*_*
+- *_kernel.sem_*
+- *_fs.mqueue.*_*
+- *_net.*_*
+
+Sysctls which are not namespaced are called _node-level_ and must be set
+manually by the cluster administrator, either by means of the underlying Linux
+distribution of the nodes (e.g., via *_/etc/sysctls.conf_*) or using a DaemonSet
+with privileged containers.
+
+[NOTE]
+====
+It is good practice to consider nodes with special sysctl settings as tainted
+within a cluster, and only schedule pods onto them which need those sysctl
+settings. It is suggested to use the
+link:http://kubernetes.io/docs/user-guide/kubectl/kubectl_taint/[Kubernetes
+_taints and toleration_ feature] to implement this.
+====
+
+[[safe-vs-unsafe-sysclts]]
+== Safe vs Unsafe Sysctls
+
+Sysctls are grouped into _safe_ and _unsafe_ sysctls. In addition to proper
+namespacing, a safe sysctl must be properly isolated between pods on the same
+node. This means that setting a safe sysctl for one pod:
+
+- must not have any influence on any other pod on the node,
+- must not allow to harm the node's health, and
+- must not allow to gain CPU or memory resources outside of the resource limits of
+a pod.
+
+By far, most of the namespaced sysctls are not necessarily considered safe.
+
+For {product-title} 3.3.1, the following sysctls are supported (whitelisted) in
+the safe set:
+
+- *_kernel.shm_rmid_forced_*
+- *_net.ipv4.ip_local_port_range_*
+
+This list will be extended in future versions when the kubelet supports better
+isolation mechanisms.
+
+All safe sysctls are enabled by default. All unsafe sysctls are disabled by
+default and must be allowed manually by the cluster administrator on a per-node
+basis. Pods with disabled unsafe sysctls will be scheduled, but will fail to
+launch.
+
+[WARNING]
+====
+Due to their nature of being unsafe, the use of unsafe sysctls is
+at-your-own-risk and can lead to severe problems like wrong behavior of
+containers, resource shortage, or complete breakage of a node.
+====
+
+[[enabling-unsafe-sysctls]]
+== Enabling Unsafe Sysctls
+
+With the warning above in mind, the cluster administrator can allow certain
+unsafe sysctls for very special situations, e.g., high-performance or real-time
+application tuning.
+
+If you want to use unsafe sysctls, cluster administrators must enable them
+individually on nodes:
+
+. Use the `*kubeletArguments*` field in the *_/etc/origin/node/node-config.yaml_*
+file, as described in
+xref:../admin_guide/manage_nodes.adoc#configuring-node-resources[Configuring Node Resources], to set the desired unsafe sysctls:
++
+----
+kubeletArguments:
+  experimental-allowed-unsafe-sysctls:
+    - "kernel.msg*,net.ipv4.route.min_pmtu"
+----
+
+. Restart the node service to apply the changes:
++
+----
+# systemctl restart atomic-openshift-node
+----
+
+Only namespaced sysctls can be enabled this way.
```

Comment:
I feel like this should be at the top of the procedure, not an afterthought at the end.